news/posts/DoodleBUGS-Introduction/index.qmd

```diff
@@ -0,0 +1,252 @@
+---
+title: "Introducing DoodleBUGS: a Browser-Based Graphical Interface for JuliaBUGS"
+description: "Shravan Goswami's GSoC 2025 final report: goals, architecture, progress vs proposal, and how to try it."
+categories:
+  - GSoC
+  - Blog
+author:
+  - name: Shravan Goswami
+    url: https://shravangoswami.com/
+date: 2025-08-28
+aliases:
+  - /news/posts/doodlebugs-gsoc-2025-final-report
+  - /news/posts/doodlebugs-gsoc-2025
+  - /news/posts/doodlebugs-introduction
+  - /news/posts/shravan-gsoc-2025-doodlebugs
+  - /doodlebugs-gsoc-2025-final-report
+  - /doodlebugs-gsoc-2025
+  - /doodlebugs-introduction
+  - /shravan-gsoc-2025-doodlebugs
+  - /news/posts/2025-08-28-DoodleBUGS-Introduction
+  - /DoodleBUGS
+bibliography: references.bib
+csl: university-of-york-ieee.csl
+link-citations: true
+nocite: |
+  @*
+---
+
+## TL;DR
+
+- DoodleBUGS is a browser-based graphical editor for Bayesian models that utilize JuliaBUGS for BUGS model compilation and connect to Julia inference backends (e.g., AdvancedHMC via AbstractMCMC).
```

Comment:
Better to have a gentle sentence on BUGS and JuliaBUGS, then say DoodleBUGS is a graphical interface for JuliaBUGS